pkg/docsonnet/load.go

```diff
@@ -71,40 +110,40 @@ func Transform(data []byte) (*Package, error) {
 	return &p, nil
 }
 
-// importer wraps jsonnet.FileImporter, to statically provide load.libsonnet,
-// bundled with the binary
-type importer struct {
-	fi   jsonnet.FileImporter
-	util jsonnet.Contents
+// newVM sets up the Jsonnet VM with the importer that statically provides doc-util.
+func newVM(mainFName string, jpaths []string) *jsonnet.VM {
+	vm := jsonnet.MakeVM()
+	vm.Importer(&jsonnet.FileImporter{JPaths: jpaths})
```

Comment:
This importer change was necessary to avoid name clashes with `./render.libsonnet`.

Specifically, with the old importer, supporting `render.libsonnet` meant capturing imports of the form `./render.libsonnet` (which is used in `doc-util/main.libsonnet`) and routing it through the embedded filesystem.

This meant that all relative package imports for `render.libsonnet` files would go through the embedded FS, which can cause problems if there is a package being `docsonnet`-ed with a `render.libsonnet` file.